qemux86-64.coffee

```diff
@@ -0,0 +1,34 @@
+deviceTypesCommon = require 'resin-device-types/common'
+{ networkOptions, commonImg, instructions } = deviceTypesCommon
+
+module.exports =
+	slug: 'qemux86-64'
+	name: 'QEMU X86 64bit'
+	arch: 'amd64'
+	state: 'experimental'
+
+	instructions: commonImg.instructions
```

Comment:
What kind of post provisioning instructions? This target boots running a script. What I miss here are the actual instruction on what to run and how.